CONTRIBUTING.md

```diff
@@ -61,6 +61,19 @@ tabs). Format your code assuming it will be read in a window 100
 columns wide. Wrap code at 100 characters and comments at 80 unless doing so
 makes the code less legible.
 
+When wrapping function signatures that do not fit on one line,
+put the name, arguments and return types on separate lines, with the closing `)`
+at the same indentation as `func`. Example:
```

Comment:
is it worse explaining motivation? e.g.
`(this helps visual separate the indented arguments from the indented function body).`